azerothcore/azerothcore-wotlk
# fix(Scripts/HallsOfReflection): Improve Halls of Reflection Events (Horde)

## Changes Proposed:
-  Organize some events
-  Gesticulation of npcs that were standing still
- Some WayPoints sniffed
- spells captured with Sniffing

## Issues Addressed:

- Closes 

## SOURCE:

https://youtu.be/YuifODRzKws

## Tests Performed:

- Compiles without error, tested on Windows 11
- 


## How to Test the Changes:


1. go to Halls of Reflection (Horde)
2. see the events
3. finish a dungeon

## Known Issues and TODO List:


- [ yes, I ask for your help to finish the alliance side and fix some things on the horde side that are still missing. for now it's the best version we have of the dungeon (horde side).]


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
